### PR TITLE
Fix incorrect assertions used

### PR DESCRIPTION
**Pull request by wendigo**
## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by losipiuk — APPROVED** [review]

File: client/trino-cli/src/test/java/io/trino/cli/TestFormatUtils.java [hunk 1]
```diff
@@ -17,472 +17,477 @@
...
+        assertThat(FormatUtils.formatCountRate(360000.0D, Duration.valueOf("10h"), false)).isEqualTo("10");
+        assertThat(FormatUtils.formatCountRate(360000.0D, Duration.valueOf("10h"), true)).isEqualTo("10/s");
+
+        assertThat(FormatUtils.formatCountRate(1.0D, Duration.valueOf("1d"), false)).isEqualTo("0");
+        assertThat(FormatUtils.formatCountRate(1.0D, Duration.valueOf("1d"), true)).isEqualTo("0/s");
+        assertThat(FormatUtils.formatCountRate(86400.0D, Duration.valueOf("1d"), false)).isEqualTo("1");
+        assertThat(FormatUtils.formatCountRate(86400.0D, Duration.valueOf("1d"), true)).isEqualTo("1/s");
+        assertThat(FormatUtils.formatCountRate(864000.0D, Duration.valueOf("1d"), false)).isEqualTo("10");
+        assertThat(FormatUtils.formatCountRate(864000.0D, Duration.valueOf("1d"), true)).isEqualTo("10/s");
+        assertThat(FormatUtils.formatCountRate(86400.0D, Duration.valueOf("10d"), false)).isEqualTo("0");
```
Comment (losipiuk) [inline client/trino-cli/src/test/java/io/trino/cli/TestFormatUtils.java:142]: can you static import FormatUtils?

Comment (wendigo) [reply client/trino-cli/src/test/java/io/trino/cli/TestFormatUtils.java:142]: yes

**Review by mosabua — APPROVED** [review]
lgtm .. thank you so much

**Comment by metadaddy** [comment]
Thanks for following up on this, folks, and thanks for the help, @wendigo!

**Comment by wendigo** [comment]
@metadaddy no worries :) Logical merge conflicts like that happen on the time! Thanks for the contribution

**Pull request merged**